src/main/java/tinkerer/controllers/GitHubHooksController.java

```diff
@@ -1,17 +1,18 @@
 package tinkerer.controllers;
 
 import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
```

Comment:
A comment as a single comment, should work! But I won't see this comment: I get the URL though